ckanext/canada/templates/public/scheming/package/read.html

```diff
@@ -1,3 +1,14 @@
 {% ckan_extends %}
 
 {% block package_additional_info %}{% endblock %}
+
+{% block actions_content %}
+  <ul class="nav nav-pills">
```

Comment:
If you want your buttons to work off the bat, change the classes here to `list-unstyled list-inline`